src/clr-addons/multilingual/abstract-multilingual.ts

```diff
@@ -3,48 +3,42 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-
-import { ElementRef, HostBinding, Injector, Input, ViewChild, Directive } from '@angular/core';
 import { ClrAbstractFormComponent } from '../abstract-form-component/abstract-form-component';
+import { Component, ElementRef, HostBinding, Injector, Input, ViewChild } from '@angular/core';
 
-@Directive()
+@Component({ template: '' })
 export abstract class ClrMultilingualAbstract extends ClrAbstractFormComponent {
   @HostBinding('class.clr-multilingual') multi = true;
 
   @Input('clrControlClasses') controlClasses = 'clr-col-md-10';
   @Input('clrSelectedLang') selectedLang: string;
   @Input() readonly: string;
+  /** Show language selector when only one language provided */
+  @Input() showSingleLanguageSelector: boolean;
```

Comment:
pls use clarity naming conventions like above (prefix with "clr")

Reply:
Done in next commit